server/calendar/src/scraping/continous_scraping.rs

```diff
@@ -3,32 +3,25 @@ use crate::scraping::task::ScrapeRoomTask;
 use crate::scraping::tumonline_calendar::{Strategy, XMLEvents};
 use crate::utils;
 use crate::utils::statistics::Statistic;
-use actix_web::web::Data;
 use awc::{Client, Connector};
-use chrono::{NaiveDate, NaiveDateTime, Utc};
+use chrono::{DateTime, NaiveDate, Utc};
 use diesel::prelude::*;
 use futures::future::join_all;
 use log::{info, warn};
 use std::time::{Duration, Instant};
-use tokio::sync::Mutex;
 use tokio::time::sleep;
 
-const SECONDS_PER_DAY: u64 = 60 * 60 * 24;
-pub async fn start_scraping(last_sync: Data<Mutex<Option<NaiveDateTime>>>) {
-    let mut interval = actix_rt::time::interval(Duration::from_secs(SECONDS_PER_DAY)); //24h
+pub async fn start_scraping(scrape_every: Duration, duration: chrono::Duration) {
```

Comment:
```suggestion
pub async fn start_scraping(scrape_every: Duration, time_window: chrono::Duration) {
```
Here and further below I think this is more clear than just duration (I was first confused)